tests/DCPS/Thrasher/ParticipantTask.cpp

```diff
@@ -156,12 +160,16 @@ ParticipantTask::svc()
     OpenDDS::DCPS::DataWriterImpl* impl =
       dynamic_cast<OpenDDS::DCPS::DataWriterImpl*>(writer.in());
 
+    const bool wait_early = durable_ ? rand() % 2 == 0 : true;
```

Comment:
Using `rand()` here may be more trouble than it's worth.  If we go with `rand()`, log the seed value used and provide a way to override it in order to have repeatable tests.